core/src/components/badge/badge.tsx

```diff
@@ -89,6 +95,7 @@ export class Badge implements ComponentInterface {
           [theme]: true,
           [`badge-${shape}`]: shape !== undefined,
           [`badge-${size}`]: size !== undefined,
+          [`badge-${this.vertical}`]: this.vertical !== undefined,
```

Comment:
```suggestion
          [`badge-vertical-${this.vertical}`]: this.vertical !== undefined,
```

Update to match the [fab naming convention](https://github.com/ionic-team/ionic-framework/blob/cd5c27a12a03cc11da7ae86d89ad402343fed1c0/core/src/components/fab/fab.tsx#L84-L85) OR we can handle this as part of `ROU-11329` if you want to add a TODO here since shape & size are also different.

Reply:
I will do a commit, as it needs update on the scss as well :)

Reply:
Yeah I was too lazy to request changes on all of them 😝